templates/README.tpl.md

```diff
@@ -1,203 +0,0 @@
-**VARS TO REPLACE IN TEMPLATE:**
```

Comment:
This is not the purpose of [cozy-template](https://github.com/cozy/cozy-template)?